docs/how/updating-datahub.md

```diff
@@ -22,6 +22,7 @@ This file documents any backwards-incompatible changes in DataHub and assists pe
 
 - Protobuf CLI will no longer create binary encoded protoc custom properties. Flag added `-protocProp` in case this 
   behavior is required.
+- `datahub get ...` CLI output format changed a bit. To use the older output format pass `datahub get --no-details ...`
```

Comment:
Is there a downside to leaving the default to be the concise (--no-details) option? That way, we don't unintentionally break anyone who might be using this as an unofficial contract.